README.md

```diff
@@ -234,6 +239,26 @@ mappings:
     provider: "$1"
 ```
 
+Glob matching offers the best performance for common mappings.
+There are however pathological cases like the following matches:
+
+    a.*.*.*.*
+    a.b.*.*.*
+    a.b.c.*.*
+    a.b.c.d.*
+
+Optimize these mappings by reversing the order, or by disabling mapping ordering.
+With unordered mapping, at each hierarchy level the most specific match wins.
```

Comment:
This is confusing. Is the example order optimal or is the reverse of the example optimal?

Reply:
It'll be optimal either if we turn off ordering (set `glob_disable_ordering` to true, which in case user doesn't care about order at all), or keep `glob_disable_ordering` being false but reorder the rules in reverse order.

Reply:
thanks, I changed it to make a positive recommendation instead of a negative example. This all makes me think we should maybe take away some options and simplify things …